service/src/main/java/uk/gov/hmcts/reform/fpl/enums/MandatoryDocuments.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum MandatoryDocuments {
+    SOCIAL_WORK_CHRONOLOGY("socialWorkChronologyDocument", "1"),
```

Comment:
Really minor, but to we need "Document" suffix her? it's added against all of those so maybe not?

Reply:
Yes. It's used as a key to match the property path, this key should match the name we've used in CaseData to describe individual documents.